lib/model/model_test.go

```diff
@@ -947,11 +947,13 @@ func TestIgnores(t *testing.T) {
 
 	db := db.OpenMemory()
 	m := NewModel(defaultConfig, protocol.LocalDeviceID, "device", "syncthing", "dev", db, nil)
-	m.AddFolder(defaultFolderConfig)
-	m.StartFolder("default")
 	m.ServeBackground()
+	m.cfg.SetFolder(defaultFolderConfig)
 	defer m.Stop()
 
+	// wait for goroutine wrapper.notifyListener to finish
+	time.Sleep(time.Duration(100) * time.Millisecond)
```

Comment:
Maybe, locally it works. I have just grown vary of timing issues and therefore like to be especially prudent - lets give it a spin on jenkins.